contrib/pkg/utils/aws/aws.go

```diff
@@ -39,3 +44,27 @@ func GetAWSCreds(credsFile, defaultCredsFile string) (string, string, error) {
 	}
 	return accessKeyIDValue.String(), secretAccessKeyValue.String(), nil
 }
+
+// ConfigureCreds loads a secret designated by the environment variables CLUSTERDEPLOYMENT_NAMESPACE
+// and CREDS_SECRET_NAME and configures AWS credential environment variables and config files
+// accordingly.
+func ConfigureCreds(c client.Client) {
+	credsSecret := utils.LoadSecretOrDie(c, "CREDS_SECRET_NAME")
+	if credsSecret == nil {
+		return
+	}
+	// Should we bounce if any of the following already exist?
```

Comment:
I think this could become whack-a-mole as creds evolve per cloud if we did bounce